common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -332,7 +332,7 @@ export interface ISequencedDocumentAugmentedMessage extends ISequencedDocumentMe
 
 /**
  * Common interface between incoming and outgoing signals.
- * @public
+ * @internal
```

Comment:
Can we move all these internal? I suspect not.

Reply:
I don't think you can even make ISignalMessageBase internal since ISignalMessage extends it (changing ISignalMessageBase would implicitly change a public interface, so I think it must be public too).